### DISCUSSION TITLE
[求助]成为传奇绳匠之后委托比以往更少了，有其他传奇绳匠来为我答疑解惑吗？

### DISCUSSION THREAD

**Discussion by kano0x0** [share121/inter-knot#462] [其他]
成为传奇绳匠之后委托比以往更少了，有其他传奇绳匠来为我答疑解惑吗？
最近有一位只花了1个月就从新手绳匠变成传奇绳匠，请问大佬有什么接更多委托的秘诀吗
温饱已经成问题了

**Comment by lllliufeng** [comment]
人家是用了什么脚本接委托吧...
